onnxruntime/python/tools/bert/bert_model_optimization.py

```diff
@@ -35,6 +35,10 @@
 from BertOnnxModelTF import BertOnnxModelTF
 from BertOnnxModelKeras import BertOnnxModelKeras
 
+# When we run this script in two cells of a Juypter Notebook, logging of first cell might overflow to the second cell.
+# Reset logger handlers as work around: https://github.com/ipython/ipython/issues/8282
+logger = logging.getLogger()
+logger.handlers = []
 logger = logging.getLogger('')
```

Comment:
Is this overwriting the above logger?

Reply:
Both getLogger() and getLogger('') will return the same logger instance (root logger). It is like singleton.

I updated the logic to avoid adding new handler instead of resetting handlers.